### PR TITLE
CB2-1718 Fixes for the pipeline process

### PR DESCRIPTION
**Pull request by Ian-Hodges**

### CODE REVIEW

File: src/wms/Database.ts [hunk 1]
```diff
@@ -49,6 +51,9 @@ export class Database {
       .where('event_date', '=', dateformat(exportDate, 'yyyy-mm-dd'));
       // TODO: add filtrering for 5 vtfs
       // .havingIn('ngt_site.site_id', [])
+
+    console.info('getstaffSchedules ending');
```
Comment (aaron-downing) [inline src/wms/Database.ts:55]: maybe say 'getstaffSchedules successful'? is there a case where this method can fail?

File: src/wms/ExportEvents.ts [hunk 2]
```diff
@@ -31,5 +33,7 @@ export async function getEvents(exportDate: Date): Promise<FacillitySchedules[]>
     throw error;
   }
 
+  console.info('getEvents ending');
```
Comment (aaron-downing) [inline src/wms/ExportEvents.ts:36]: again maybe successful instead of ending.

File: package.json [hunk 3]
```diff
@@ -34,13 +34,10 @@
     "security-checks": "git secrets --scan && git log -p | scanrepo",
     "clean": "rimraf ./.build ./.artifact ./.serverless ./.webpack ./*.zip",
     "compile": "tsc",
-    "build:dev": "cross-env NODE_ENV=local serverless webpack",
-    "build:production": "source config/.env.production && cross-env NODE_ENV=production webpack -c ./scripts/webpack.config.build.js --env=production",
-    "build": "npm-run-all clean build:dev",
+    "build": "node_modules/typescript/bin/tsc --rootDir ./ --outDir .build --sourceMap false && npm run build:copy && rimraf .build/tests",
```
Comment (premghinde) [inline package.json:37]: Can't this be just `tsc`?

File: scripts/webpack.config.build.js [hunk 4]
```diff
@@ -2,16 +2,16 @@ const path = require('path');
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
 // const nodeExternals = require('webpack-node-externals');
 
-const { NODE_ENV, SERVICE, AWS_PROVIDER_REGION, AWS_PROVIDER_STAGE } = process.env;
+const { NODE_ENV, SERVICE, AWS_REGION, AWS_PROVIDER_STAGE } = process.env;
```
Comment (premghinde) [inline scripts/webpack.config.build.js:5]: delete this file?

File: scripts/webpack.config.js [hunk 5]
```diff
@@ -3,10 +3,10 @@ const slsw = require('serverless-webpack');
 const nodeExternals = require('webpack-node-externals');
 const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
 
-const { NODE_ENV, SERVICE, AWS_PROVIDER_REGION, AWS_PROVIDER_STAGE } = process.env;
+const { NODE_ENV, SERVICE, AWS_REGION, AWS_PROVIDER_STAGE } = process.env;
```
Comment (premghinde) [inline scripts/webpack.config.js:6]: delete this file?

File: src/wms/Database.ts (hunk 1 above)
Comment (Ian-Hodges) [reply src/wms/Database.ts:55]: You could get some events that have failed. I'm happy with 'ending' which doesn't indicate any success or failure. The purpose of the log entry is to indicate that the method has run and is about to exit. Handy to determine where any timeouts occur.

File: src/wms/ExportEvents.ts (hunk 2 above)
Comment (Ian-Hodges) [reply src/wms/ExportEvents.ts:36]: I'm happy with 'ending' which doesn't indicate any success or failure. The purpose of the log entry is to indicate that the method has run and is about to exit. Handy to determine where any timeouts occur.

File: src/wms/Database.ts (hunk 1 above)
Comment (premghinde) [reply src/wms/Database.ts:55]: Is it worth having the method in a try catch so you can handle the failure scenario?

File: scripts/webpack.config.build.js (hunk 4 above)
Comment (Ian-Hodges) [reply scripts/webpack.config.build.js:5]: Very likely going to be deleted. There is a ticket next sprint to clean up redundant packages and config.

File: scripts/webpack.config.js (hunk 5 above)
Comment (Ian-Hodges) [reply scripts/webpack.config.js:6]: Very likely going to be deleted. There is a ticket next sprint to clean up redundant packages and config.

File: package.json (hunk 3 above)
Comment (Ian-Hodges) [reply package.json:37]: Works locally fine. Testing it in the pipeline...

File: src/wms/Database.ts (hunk 1 above)
Comment (Ian-Hodges) [reply src/wms/Database.ts:55]: Failure to read the database is terminal and is captured in the handler try/catch. It does raise the question about whether we want a retry policy. I'll add a ticket for that.

File: package.json (hunk 3 above)
Comment (Ian-Hodges) [reply package.json]: Works fine in pipeline too. Change made.

**Review by danielchildsbjss — APPROVED** [review]

**Review by aaron-downing — APPROVED** [review]

**Pull request merged**